From 390499cb4ac9bb04e5161475a4b934a2b405312f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sun, 20 Mar 2016 09:56:44 +0100
Subject: [PATCH] panel: Fix missing panel background

This fixes the missing background, when the built-in theme is disabled.
---
 panel/panel.vala | 1 +
 1 file changed, 1 insertion(+)

diff --git a/panel/panel.vala b/panel/panel.vala
index b1a6a0a..c7a4bf4 100644
--- a/panel/panel.vala
+++ b/panel/panel.vala
@@ -26,6 +26,7 @@ public class MainPanel : Gtk.Box
         Object(orientation: Gtk.Orientation.HORIZONTAL);
         this.intended_size = size;
         get_style_context().add_class("budgie-panel");
+        get_style_context().add_class(Gtk.STYLE_CLASS_BACKGROUND);
     }
 
     public override void get_preferred_height(out int m, out int n)
-- 
2.7.3

